seed/seed.json

```diff
@@ -323,6 +323,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "AdServingIncreaseMaximumAdNotificationsPerDayTo60ForTier1And2Regions",
```

Comment:
Does it make sense to specify `60` in the study name? We could modify the parameter in the future. 
maybe just `MaxAdNotificationsPerDayStudy`?

Reply:
fixed as requested by @tmancey